fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -915,8 +916,18 @@ class BrowserRobot {
     fun fillPdfForm(name: String) {
         // Set PDF form text for the text box
         itemWithResId("pdfjs_internal_id_10R").setText(name)
-        // Click PDF form check box
-        itemWithResId("pdfjs_internal_id_11R").click()
+        mDevice.waitForWindowUpdate(packageName, waitingTime)
+        if (itemWithResId("pdfjs_internal_id_11R").exists().not()) {
+            // Verify if keyboard is visible
+            verifyKeyboardVisibility(true)
```

Comment:
Should this check be in the conditional if above?

Reply:
Done !